pkg/rancher-components/src/components/Card/Card.vue

```diff
@@ -11,7 +12,8 @@ export default {
     },
     buttonAction: {
       type:    Function,
-      default: () => {}
+      // TODO: is this the correct default? Originally () => { }
```

Comment:
The original function is expected to do not return any value (falsy/undefined). I am not so sure if returning a value (truthy/true) may generate issues.